flycheck.el

```diff
@@ -1384,7 +1384,8 @@ to the default levels."
                           (flycheck-error-list-current-errors)))
          (levels-with-defaults (append '(info warning error) levels))
          (uniq-levels (seq-uniq levels-with-defaults))
-         (level (flycheck-completing-read prompt uniq-levels nil)))
+         (level (flycheck-completing-read prompt uniq-levels
+                                          (car uniq-levels))))
```

Comment:
This look wrong: we do want to offer an empty entry (it corresponds to setting a nil level)